cmd/ak/cmd/sessions/test.go

```diff
@@ -156,12 +163,41 @@ var testCmd = common.StandardCommand(&cobra.Command{
 		expected := strings.TrimSpace(string(a.Comment))
 		actual := strings.TrimSpace(prints.String())
 
+		var errs []error
+
 		if actual != expected {
 			edits := myers.ComputeEdits(span.URIFromPath("want"), expected, actual)
-			return errors.New(fmt.Sprint(gotextdiff.ToUnified("want", "got", expected, edits)))
+			errs = append(errs, errors.New(fmt.Sprint("output:\n", gotextdiff.ToUnified("want", "got", expected, edits))))
+		}
+
+		if expectedCallsTxt != nil {
+			results, err := sessions().GetLog(ctx, sdkservices.SessionLogRecordsFilter{
+				SessionID: sid,
+				Types:     sdktypes.CallSpecSessionLogRecordType,
+				PaginationRequest: sdktypes.PaginationRequest{
+					Ascending: true,
+				},
+			})
+			if err != nil {
+				return fmt.Errorf("get calls: %w", err)
+			}
+
+			var callsTxt strings.Builder
+			for _, r := range results.Records {
+				f, _, _ := r.GetCallSpec().Data()
+				fmt.Fprintf(&callsTxt, "%s\n", f.GetFunction().Name())
+			}
+
+			expected := strings.TrimSpace(kittehs.StringWithoutComments(string(expectedCallsTxt)))
```

Comment:
Nit: Move to a function.

Reply:
why? i'm actually fine with the way it is now.